applications/machine_learning/configuration/nrf52840dk_nrf52840/prj_nus.conf

```diff
@@ -82,7 +82,7 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 CONFIG_ISR_STACK_SIZE=1536
 CONFIG_MAIN_STACK_SIZE=1024
 CONFIG_IDLE_STACK_SIZE=512
-CONFIG_BT_RX_STACK_SIZE=1600
+CONFIG_BT_RX_STACK_SIZE=2000
```

Comment:
minor nit: This value is identical across platforms as far as I can tell.

So another option is to actually add:
```
config BT_RX_STACK_SIZE
        dafult 2000
```
to this file: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/machine_learning/Kconfig